package.json

```diff
@@ -1,6 +1,7 @@
 {
     "private": true,
     "dependencies": {
-        "typescript": "4.6.3"
+        "typescript": "4.6.3",
+        "@bazel/worker": "^5.4.2"
```

Comment:
This might be unnecessary given that this file is only used to pick the typescript version.